src/commonMain/kotlin/fr/acinq/lightning/channel/WaitForFundingSigned.kt

```diff
@@ -62,7 +61,7 @@ data class WaitForFundingSigned(
                     is Helpers.Funding.FirstCommitments -> {
                         val (signedFundingTx, commitments) = firstCommitmentsRes
                         logger.info { "c:$channelId funding tx created with txId=${commitments.fundingTxId}. ${fundingTx.localInputs.size} local inputs, ${fundingTx.remoteInputs.size} remote inputs, ${fundingTx.localOutputs.size} local outputs and ${fundingTx.remoteOutputs.size} remote outputs" }
-                        if (commitments.channelFeatures.hasFeature(Feature.ZeroConfChannels)) {
+                        if (staticParams.useZeroConf) {
```

Comment:
We need to send an alias in the `channel_ready` tlv to be able to user zero-conf.